### PR TITLE
[CNVS Upgrade] Migrate legacy components: application loading indicator

### PR DESCRIPTION
**Pull request by jfurrow**

### CODE REVIEW

File: src/styles/components/application-loading-indicator/styles.less [hunk 1]
```diff
@@ -1,25 +1,39 @@
...
-  50% {
-    opacity: 0.15;
-  }
 
-  100% {
-    opacity: 0.4;
+
+& when (@application-loading-indicator-enabled) {
+
+  @keyframes application-loading-indicator-pulse {
```
Comment (ashenden) [inline src/styles/components/application-loading-indicator/styles.less]: If you want to be consistent with all styling in CNVS, we'd have extra space after opening and before closing `{ }`.  Not a blocker, but helps with consistency.

**Comment by ashenden** [comment]
Looks good and runs.  Left you one comment on code style, but other than that, it's good to go.  Not in this branch, but we may want to consolidate "loaders" into a single component at some point in the future.

**Comment by jfurrow** [comment]
I like the idea of the `loaders` component!

File: src/styles/components/application-loading-indicator/styles.less [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+application-loading-indicator/styles.less
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+----------------------------------------------------------------------------- */
+
+
+
+& when (@application-loading-indicator-enabled) {
+
+  @keyframes application-loading-indicator-pulse {
```
Comment (rcorral) [inline src/styles/components/application-loading-indicator/styles.less]: why aren't we just calling it `pulsating-pineapple`?

Comment (jfurrow) [inline src/styles/components/application-loading-indicator/styles.less]: lol...

**Pull request merged**